src/woo_publications/conf/ci.py

```diff
@@ -25,9 +25,6 @@
 # shut up logging
 mute_logging(LOGGING)
 
-# don't spend time on password hashing in tests/user factories
-PASSWORD_HASHERS = ["django.contrib.auth.hashers.UnsaltedMD5PasswordHasher"]
```

Comment:
instead make it:

```py
PASSWORD_HASHERS = ["django.contrib.auth.hashers.MD5PasswordHasher"]
```
this one *does* still exist and is faster than all the other password hashes, which is the reason why this is specified in the CI settings: faster tests